src/table/virtualized-table/__tests__/Table.spec.tsx

```diff
@@ -87,9 +104,9 @@ describe('<Table />', () => {
   afterEach(() => jest.restoreAllMocks());
 
   it('should render', async () => {
-    renderTable();
+    await renderTable();
```

Comment:
Do we need await here because of act? if that is true, why we should await for act?

Reply:
`act` returns a Promise. So we wait for that to resolve. If not, the test would log errors telling us that wrap the call in  `act`. Don't fully understand as to why it's needed tho.

Reply:
Right, so quick rendering tells me that `await act` is needed because `Table` component does some async data fethcing in a `useEffect`.  So that should be a way to await for that data fetch to complete.